modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/jaxrs/exception/mapper/BasePriceListCannotDeleteExceptionMapper.java

```diff
@@ -39,13 +40,19 @@
 	extends BaseExceptionMapper<CommerceBasePriceListCannotDeleteException> {
 
 	@Override
-	public String getErrorDescription() {
-		return "Unable to delete base price list";
+	protected Problem getProblem(
+		CommerceBasePriceListCannotDeleteException
+			commerceBasePriceListCannotDeleteException) {
+
+		return new Problem(
+			null, Response.Status.BAD_REQUEST,
+			commerceBasePriceListCannotDeleteException.getMessage(),
+			"CommerceOrderValidatorException");
```

Comment:
why is this one a "CommerceOrderValidatorException"?

Reply:
oh nice catch, bad typo, I will fix that! thanks!